### PR TITLE
[Runtime Security-RN] O'Neal Update 3 PCCE

### PR DESCRIPTION
**Pull request by anaghapamidi**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--prisma-cloud-docs--hlxsites.hlx.page/
- After: https://<branch>--prisma-cloud-docs--hlxsites.hlx.page/
- Worker: https://prisma-cloud-docs-production.adobeaem.workers.dev/?branch=<branch>

### CODE REVIEW

File: docs/en/compute-edition/32/rn/release-information/release-notes-32-03.adoc [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+|Type
+|Maintenance Release
+
+|SHA-256 TODO:
+|a4790f3e94509fb1d80b0ff5a3567ac57295fd52b976b7f67d34ab268f9636ee
+
+|===
+
+Review the https://docs.paloaltonetworks.com/prisma/prisma-cloud/32/prisma-cloud-compute-edition-admin/install/system_requirements[system requirements] to learn about the supported operating systems, hypervisors, runtime, tools, and orchestrators.
```
Comment (smitapaloalto) [inline docs/en/compute-edition/32/rn/release-information/release-notes-32-03.adoc:24]: Please check the https://docs.paloaltonetworks.com/prisma/prisma-cloud/32/prisma-cloud-compute-edition-admin/install/system_requirements link, it throws 404 for me. I am not sure if I have insufficient permissions. Please double-check.

File: docs/en/compute-edition/32/rn/release-information/release-notes-32-03.adoc [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+toc::[]
+
+[#upgrade]
+=== Upgrade from Previous Releases
+
+[#upgrade-defender]
+==== Upgrade Defender Versions 22.06 and Earlier
+
+With the `v32.00` release, https://docs.paloaltonetworks.com/prisma/prisma-cloud/32/prisma-cloud-compute-edition-admin/welcome/support_lifecycle[Defender versions supported (n, n-1, and n-2)] are `v32.00`, `v31.00`, and `v30.00`.
```
Comment (smitapaloalto) [inline docs/en/compute-edition/32/rn/release-information/release-notes-32-03.adoc:38]: https://docs.paloaltonetworks.com/prisma/prisma-cloud/32/prisma-cloud-compute-edition-admin/welcome/support_lifecycle Please check. It throws 404 by me.

File: docs/en/compute-edition/32/rn/release-information/release-notes-32-03.adoc [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+To prepare for this update, you must upgrade your Defenders from version `v22.06` (Kepler) or earlier to a later version.
+Failure to upgrade Defenders will result in disconnection of any Defender version below `v22.12` such as `v22.06`.
+
+[#upgrade-console]
+==== Upgrade the Prisma Cloud Console
+
+With the `v32.00` release, the https://docs.paloaltonetworks.com/prisma/prisma-cloud/32/prisma-cloud-compute-edition-admin/welcome/support_lifecycle[supported Console versions (n, n-1, and n-2)] are `v32.00`, `v31.00`, and `v30.00`.
```
Comment (smitapaloalto) [inline docs/en/compute-edition/32/rn/release-information/release-notes-32-03.adoc:46]: https://docs.paloaltonetworks.com/prisma/prisma-cloud/32/prisma-cloud-compute-edition-admin/welcome/support_lifecycle Please check. It throws 404 for me.

File: docs/en/compute-edition/32/rn/release-information/release-notes-32-03.adoc (hunk 1 above)
Comment (rcaballeromx) [reply docs/en/compute-edition/32/rn/release-information/release-notes-32-03.adoc:24]: @smitapaloalto 
It appears that the redirect is broken. We need to change it to https://docs.prismacloud.io/en/compute-edition/32/admin-guide/install/system-requirements
Can you confirm that the link works?

File: docs/en/compute-edition/32/rn/release-information/release-notes-32-03.adoc (hunk 2 above)
Comment (rcaballeromx) [reply docs/en/compute-edition/32/rn/release-information/release-notes-32-03.adoc:38]: @smitapaloalto 
I suspect the same issue here. Can you confirm that this link works:
https://docs.prismacloud.io/en/compute-edition/32/admin-guide/upgrade/support-lifecycle

File: docs/en/compute-edition/32/rn/release-information/release-notes-32-03.adoc (hunk 3 above)
Comment (rcaballeromx) [reply docs/en/compute-edition/32/rn/release-information/release-notes-32-03.adoc:46]: @anaghapamidi 
I think the correct link should be:
https://docs.prismacloud.io/en/compute-edition/32/admin-guide/upgrade/support-lifecycle

**Review by rcaballeromx — CHANGES_REQUESTED** [review]
Some redirects seem not to be working properly.
I've included the correct links.

File: docs/en/compute-edition/32/rn/release-information/release-notes-32-03.adoc (hunk 1 above)
Comment (smitapaloalto) [reply docs/en/compute-edition/32/rn/release-information/release-notes-32-03.adoc:24]: The link works fine now. thanks.

File: docs/en/compute-edition/32/rn/release-information/release-notes-32-03.adoc (hunk 3 above)
Comment (smitapaloalto) [reply docs/en/compute-edition/32/rn/release-information/release-notes-32-03.adoc:46]: The link works fine now. Thanks.

File: docs/en/compute-edition/32/rn/release-information/release-notes-32-03.adoc (hunk 1 above)
Comment (smitapaloalto) [reply docs/en/compute-edition/32/rn/release-information/release-notes-32-03.adoc:24]: It works fine. Thanks.

**Comment by anaghapamidi** [comment]
@rcaballeromx - Links are updated.
cc: @manukumar6

**Review by manukumar6 — APPROVED** [review]

**Pull request merged**